tests/framework/CodeInterfaceTests/raven_running_raven_internal_models/ext_dataobjects.xml

```diff
@@ -0,0 +1,34 @@
+<DataObjects>
+	<PointSet name="inputHolder">
+		<Input>DeltaTimeScramToAux,DG1recoveryTime</Input>
+		<Output>OutputPlaceHolder</Output>
+	</PointSet>
+	<PointSet name="Pointset_from_database_for_rom_trainer">
+		<Input>DeltaTimeScramToAux,DG1recoveryTime</Input>
+		<Output>CladTempThreshold,time</Output>
+	</PointSet>
+	<HistorySet name="Historyset_from_database_for_rom_trainer">
+		<Input>DeltaTimeScramToAux,DG1recoveryTime</Input>
+		<Output>CladTempThreshold</Output>
+	</HistorySet>
+	<PointSet name="data_for_sampling_empty_at_begin">
+		<Input>DeltaTimeScramToAux,DG1recoveryTime</Input>
+		<Output>OutputPlaceHolder</Output>
+	</PointSet>
+	<PointSet name="data_for_sampling_empty_at_begin_nd">
+		<Input>DeltaTimeScramToAux,DG1recoveryTime</Input>
+		<Output>OutputPlaceHolder</Output>
+	</PointSet>
+	<PointSet name="outputMontecarloRom">
+		<Input>DeltaTimeScramToAux,DG1recoveryTime</Input>
+		<!--Output>CladTempThreshold</Output-->
```

Comment:
I think this is going to create a "validation problem" of the XSD...

Reply:
I do not remember if we changed the XSD schema for the DataObjects in order to accept optional <Input> and <Output>

Reply:
It passed validation on my machine and on the qsub tests, so it seems okay.

The XSD shows minOccurs=0 for both Input and Output nodes.

Reply:
Ok.
We modified it. I do not like it too much but I think it is a limitation of the XSD schema now...So nothing we can do :) 

Approved